recipes/pcapplusplus/all/patches/0003-Use-pthreads4w.patch

```diff
@@ -0,0 +1,18 @@
+--- mk/vs/PcapPlusPlusPropertySheet.props.template
```

Comment:
Just FYI: v22.05 is the last version that needs `pthreads`, from the next version PcapPlusPlus will move to C++11 and will use the built-in threads in the std library

Reply:
Yes, I am aware of that, but pcapplusplus version with C++11 has not been released yet.

Do you have plans to release a version soon?

Reply:
Probably not very soon, but hopefully in the coming months, please see my comment here: https://github.com/conan-io/conan-center-index/pull/12900#discussion_r973944220